gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -50,6 +38,8 @@
 import java.util.Map;
 import java.util.Objects;
 
+import static com.google.gson.Gson.*;
```

Comment:
Wildcard imports [should be avoided](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)